django/applications/catmaid/management/commands/catmaid_cache_update_worker.py

```diff
@@ -55,7 +55,7 @@ def update(self, updates, cursor):
             if g.n_last_edited_skeletons_limit:
                 params['n_last_edited_skeletons_limit'] = int(g.n_last_edited_skeletons_limit)
             if g.hidden_last_editor_id:
-                param['hidden_last_editor_id'] = int(g.hidden_last_editor_id)
+                params['hidden_last_editor_id'] = int(g.hidden_last_editor_id)
```

Comment:
seemed an obvious fix. I think we've seen this a few times so far, so there may be some broader pattern to grep for in the sources that got cut'n'pasted all over?

Reply:
Yep, thanks. It could also just be a simple typo in not often used functionality. :grin:  Using a regex to find this pattern though is a good idea. This looks for `param[` locations that have `params` in the 15 lines before it and it seems to work for the location you found (when in catmaid django app directory):
```
pcregrep -M  'params(.*\n){0,15}.*param\[' management/commands/catmaid_cache_update_worker.py
```
And this might have been the last occurence of this particular pattern (`params` vs `param`), because the following doesn't finds any other location: `pcregrep -M  'params(.*\n){0,15}.*param\[' **/*.py`.